config.py

```diff
@@ -36,7 +36,7 @@ class Config(object):
     JSON_SECRET_KEYS = os.getenv('JSON_SECRET_KEYS')
 
     REDIS_HOST = os.getenv('REDIS_HOST', 'localhost')
-    REDIS_PORT = os.getenv('REDIS_PORT', 6379)
+    REDIS_PORT = os.getenv('REDIS_PORT', 7379)
```

Comment:
I can't remember why, but I think this change can't be checked in as something else uses it?  Could be wrong though

Reply:
I had to change it to run the tests and forgot to revert back to the original. Will revert to the original.